index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE>
```

Comment:
Your HTML is nicely written and laid out, and you are using comments well.

Just double-check your doctype declaration here - you may run into issues when you are trying to validate your HTML. Compare with here: https://www.w3schools.com/tags/tag_doctype.asp